script/signup.js

```diff
@@ -24,17 +27,25 @@ function emailCheck(e) {
     } else if (!checkEmail(input.value)) {
       createTag(input, "올바른 이메일이 아닙니다.");
     } else {
-      const user = emailDuplicateCheck(input.value);
-      if (user) {
-        createTag(input, "이미 사용중인 이메일입니다.");
-      }
+      const user = fetch(`https://bootcamp-api.codeit.kr/api/check-email`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ email: input.value }),
+      }).then((response) => {
+        if (response.status === 409) {
+          createTag(input, "이미 사용중인 이메일입니다.");
+        }
+      });
       emailEnable = true;
     }
   }
 }
 
 function PasswordCheck(e, type) {
   //패스워드 사용 가능 여부 판단 함수
+  const MIN_PASSWORD = 9;
```

Comment:
상수화 좋네요 👍 

의미가 좀더 잘 전달되도록 아래처럼 명명해보셔도 좋을거 같아요
```suggestion
  const MIN_PASSWORD_LENGTH = 9;
```